### PR TITLE
add scikit-rf

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #2323

Ping @arsenovic. I'm not sure why your PR still tried to pull `numpy 1.9` This one fixes some minor issues with your recipe (probably unrelated to the failure there) and it seems to be working, Fingers crossed.

### CODE REVIEW

File: recipes/scikit-rf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - skrf
+    - skrf.calibration
+    - skrf.data
+    - skrf.io
+    - skrf.media
+
+about:
+  home: http://www.scikit-rf.org
+  license: BSD 3-Clause
```
Comment (synapticarbors) [inline recipes/scikit-rf/meta.yaml]: Please add `license_file: LICENSE.txt`.

Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: There is no license file. It will be added once it is fixed upstream.

Comment (synapticarbors) [inline recipes/scikit-rf/meta.yaml]: My mistake. I misread the manifest file. Also it looks like 0.14.5 that was just released, although that does not fix the license.

Comment (arsenovic) [inline recipes/scikit-rf/meta.yaml]: @ocefpaf, thanks !
 sorry but it's not clear to me if you are asking me to do this, and if so  which repo i need to fork to do this?

Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: > sorry but it's not clear to me if you are asking me to do this, and if so which repo i need to fork to do this?

No need for you to do anything here. The upstream library needs a license file that we can add here.

Comment (arsenovic) [inline recipes/scikit-rf/meta.yaml]: ok, fyi the scikit-rf licence is in [skrf/LICENSE.txt ](https://github.com/scikit-rf/scikit-rf/blob/master/LICENSE.txt)

Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: But not in the source distro tarball :unamused: 

You can send a PR upstream to add it and, once a new release is available fix it in the feedstock. :wink:

Comment (arsenovic) [inline recipes/scikit-rf/meta.yaml]: OK, one last question: to update to the next  version 0.14.5, i should personally commit to 
https://github.com/conda-forge/scikit-rf-feedstock?

Comment (synapticarbors) [inline recipes/scikit-rf/meta.yaml]: You should fork the feedstock repo and make a pull request. See full details here:

https://github.com/conda-forge/scikit-rf-feedstock#updating-scikit-rf-feedstock

**Pull request merged**